docs/usage.md

```diff
@@ -32,23 +32,22 @@ c2pa = { version = "0.45.2", features = ["file_io", "add_thumbnails"] }
 
 ## Features
 
-The Rust library crate provides the following capabilities:
-
-* `openssl` *(enabled by default)* - Enables the system `openssl` implementation for cryptography.
-* `rust_native_crypto` - Enables the Rust native implementation for cryptography.
-* `add_thumbnails` generates thumbnails automatically for JPEG and PNG files. (no longer included with `file_io`)
-* `fetch_remote_manifests` enables the verification step to retrieve externally referenced manifest stores.  External manifests are only fetched if there is no embedded manifest store and no locally adjacent .c2pa manifest store file of the same name.
-* `file_io` enables manifest generation, signing via OpenSSL, and embedding manifests in [supported file formats](supported-formats.md).
-* `json_schema` is used by `make schema` to produce a JSON schema document that represents the `ManifestStore` data structures.
-* `serialize_thumbnails` includes binary thumbnail data in the [Serde](https://serde.rs/) serialization output.
```

Comment:
`serialize_thumbnails` has been removed also very recently - does it warrant the same note as the v1_apis? Or do the v1_apis cover that feature?